themes/classic/templates/catalog/_partials/sort-orders.tpl

```diff
@@ -23,7 +23,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  *}
 <span class="col-sm-3 col-md-3 hidden-sm-down sort-by">{l s='Sort by:' d='Shop.Theme.Global'}</span>
-<div class="{if !empty($listing.rendered_facets)}col-sm-9 col-xs-8{else}col-sm-12 col-xs-12{/if} col-md-9 products-sort-order dropdown">
+<div class="{if !empty($listing.rendered_facets)}col-sm-9 col-8{else}col-12{/if} col-md-9 products-sort-order dropdown">
```

Comment:
Missing `col-sm-12` after `{else}`

Reply:
`col-sm-12` is useless because there is `col-12` which is also 100% width
cf the doc here : https://v4-alpha.getbootstrap.com/layout/grid/